components/_header.scss

```diff
@@ -17,12 +16,10 @@ header {
 
 .tws-logo-type {
   font-family: "Nanum Pen Script";
```

Comment:
When logo finalized, we'll make it a png and upload. WIP at the moment. Should be ready by later tn